setup.py

```diff
@@ -76,6 +76,7 @@
 
 # Add the project-global data
 package_info['package_data'].setdefault(PACKAGENAME, [])
+package_info['package_data'][PACKAGENAME].append('*/tests/data/*')
```

Comment:
We seem to have a lot of locations for test data. Do you think it might make sense to consolidate them?